example_7/reference_generator/send_trajectory.cpp

```diff
@@ -96,6 +96,17 @@ int main(int argc, char ** argv)
     trajectory_msg.points.push_back(trajectory_point_msg);
   }
 
+  // send zero velocities in the end
+  std::memset(
+    trajectory_point_msg.velocities.data(), 0.0,
+    trajectory_point_msg.velocities.size() * sizeof(double));
```

Comment:
```suggestion
  std::fill(trajectory_point_msg.velocities.begin(), trajectory_point_msg.velocities.end(), 0.0);
```

Considering that this may be code that people will copy around, I'd really opt for something that doesn't suggest writing raw memory:
https://stackoverflow.com/questions/8848575/fastest-way-to-reset-every-value-of-stdvectorint-to-0

Reply:
good point, done!